7_drivers_munge/src/GCM_driver_utils.R

```diff
@@ -28,8 +28,8 @@ reconstruct_gcm_grid <- function(grid_params) {
 #' @param tile_dim single numeric value representing how many grid cells to
 #' group into a single tile. NOTE - sf_make_grid() can only make square grid
 #' polygons. The GCM grid is 217 cells wide by 141 high. With a `tile_dim` of
-#' 10 (tiles = 10 grid cells x 10 grid cells), tiles won't cover full height
-#' of the GCM grid and there will be 7 columns left out on the right and 1 left
+#' 15 (tiles = 15 grid cells x 15 grid cells), tiles won't cover full height
```

Comment:
glad this spend things up! Perhaps when we go multi-state we could get away with 20?

Reply:
Yes, I think so! I tested 20 cells per tile in #298 but it didn't make any difference for MN, so I just left as-is. I believe I left a not about increasing tile size when we scale up in #279